src/php/lib/Grpc/BaseStub.php

```diff
@@ -324,25 +473,14 @@ protected function _serverStreamRequest($method,
      *
      * @return BidiStreamingCall The active call object
      */
-    protected function _bidiRequest($method,
+    protected function _bidiRequest(
+        $method,
```

Comment:
formatting: sounds like the formatting commit has made this worse?

Reply:
I'm new to the php format, like it uses 4 space. So I use a tool called php-cs-format Stanley suggests.
Maybe the new rule of formatting does look like this.